domains/runtime/core-eth-relay/Cargo.toml

```diff
@@ -71,6 +76,7 @@ std = [
 	"pallet-transaction-payment/std",
 	"pallet-transporter/std",
 	"scale-info/std",
+	"snowbridge-ethereum-beacon-client/std",
```

Comment:
Why no `snowbridge-beacon-primitives` though?

Reply:
I missed it since at present it's `std` is not used here. Added it.